Scripts/utils/freight_costs.py

```diff
@@ -0,0 +1,130 @@
+import numpy
+from typing import Dict
+
+
+def calc_road_cost(unit_costs: Dict[str, Dict],
+                   impedance: Dict[str, numpy.ndarray]):
+    """Calculate freight road costs.
+
+    Parameters
+    ----------
+    unit_costs : Dict[str, Dict]
+        Freight mode (truck/freight_train/ship) : mode
+            Mode (truck/trailer_truck) : unit cost name
+                unit cost name : unit cost value
+    impedance : Dict[str, numpy.ndarray]
+        Type (time/dist/toll) : numpy 2d matrix
+
+    Returns
+    ----------
+    road_cost : numpy.ndarray
+        impedance type cost : numpy 2d matrix
+    """
+    mode_cost = {}
+    for mode, params in unit_costs["truck"].items():
+        road_cost = (impedance["time"]*params["time_cost"]
+                     + impedance["dist"]*params["dist_cost"]
+                     + impedance["toll"]) / params["avg_load"]
```

Comment:
What I meant was something like this:
```suggestion
        road_cost = sum(impedance[k] * params[k] for for k in impedance) / params["avg_load"]
```
But that means that you would have to rename the parameters "time", "dist" and "toll". And toll parameter would obviously be 1.

Reply:
Right, we can also use comprehension style here as it would could simplify the func in case of road and rail calc. Not necessarily with ship calc. Can make a revision of this suggestion.